src/bikeracks/templates/scripts.js

```diff
@@ -596,10 +596,10 @@ BikeMap.prototype.popupContent = function(state) {
     let onlineStatus,
         voterStatus;
     if (this.auth.currentUser) {
-        onlineStatus = true;
+        onlineStatus = Boolean(this.auth.currentUser);
```

Comment:
If `Boolean(this.auth.currentUser)` is false then you won't get here. You don't need this `if` at all. You only need this line.

Reply:
Done.